cudapoa/src/utils.cu

```diff
@@ -0,0 +1,159 @@
+/*
+* Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#include "claragenomics/cudapoa/utils.hpp"
+#include "allocate_block.hpp"
+
+namespace claraparabricks
+{
+
+namespace genomeworks
+{
+
+namespace cudapoa
+{
+
+/// \brief Create a small set of batch-sizes to increase GPU parallelization.
```

Comment:
the implementation file shouldn't need doxygen docs as long as the public header has it (which I believe it does in `utils.hpp`)

Reply:
good catch, I just copied and forgot to remove these comments.